package.nls.json

```diff
@@ -10,6 +10,7 @@
 "extension.showGettingStarted":"Getting Started Guide",
 "extension.newQuery":"New Query",
 "extension.rebuildIntelliSenseCache":"Refresh IntelliSense Cache",
+"mssql.loadCompletionExtension":"Load completion extension into sql language service",

```

Comment:
if this text will be show to user we should capitalize `SQL`.

Reply:
No need to expose the command in Command Palette, removed now.